packages/schematics/angular/application/files/src/index.html.template

```diff
@@ -7,6 +7,311 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto+Mono"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+
+  <style>
+    body {
```

Comment:
The idea was to keep them in the `index` file (even though not really the way things are normally done) so that the user can easily delete all of the template code in one full swoop.